en/advanced_config/ethernet_setup.md

```diff
@@ -0,0 +1,85 @@
+# PX4 Ethernet Setup
+
+Ethernet connectivity a faster, more reliable, and more flexible communication technology than serial communication approach.
+This method is preferred over serial communication in case of direct connection between Pixhawk and IP Radio telemetry, companion computers and Networks.
+It utilizes UDP connection for MAVLINK communication over the port 14550 with [MAVLINK2](https://mavlink.io/en/guide/mavlink_2.html). 
+
+## Supported Flight Controllers
+
+PX4 supports Ethernet connectivity on [Pixhawk 5X-standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-011%20Pixhawk%20Autopilot%20v5X%20Standard.pdf) flight controllers that have an Ethernet port.
+It may also be supported on other boards.
+
+Supported flight controllers include:
+
+ - [Holybro Pixhawk 5X](../flight_controller/pixhawk5x.md)
+
+
+## Connecting PX4 and a Ubuntu Linux Computer
```

Comment:
FYI only, here I'm being very clear on what we are showing. This is not PX4 and a companion, or PX4 and Windows Computer. This matters because you use the word "PC" to refer to the computer. To me and anyone in my generation that implies a Windows computer :-)

Reply:
OK. I still have a long way to be clear. :D